docs/integrations/webhooks/events/index.mdx

```diff
@@ -444,7 +444,7 @@ Payload objects with the following scopes take the form that follows:
 
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/modifier/updated | Fires when you edit attributes for a local or shared modifier. Updates to the global store and channel locale as an override triggers the webhook.<br /><br />The `context` fields are present for only updates to overrides, not the global store. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview. |
+| store/modifier/updated | Fires when you edit attributes for a local or shared modifier. Updates to the channel locale as an override, not the global store, trigger the webhook.<br /><br />The `context` fields are present when you update the overrides. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview. |
```

Comment:
@bc-andreadao maybe we can think of a more external facing terminology here `channel locale as an override` and `global store` seems a little bit confusing for someone externally. 

Maybe something like this would be enough for now and we can skip all the rest:
`Only fires when you edit attributes for a local or shared modifier in a context of a channel and locale. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.`